### PR TITLE
Reduce dependency on concrete packages from symfony

### LINKED ISSUE #32: Don't require Symfony full stack
https://github.com/sonata-project/SonataDashboardBundle/blob/da97986ed8dcd3c85df9068c729de93e2e2eaabd/composer.json#L20

This requirement must be replaced by components you really need.

### PR DESCRIPTION
**Pull request by hason**
Fixes #32.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,12 +17,16 @@
         }
     ],
     "require": {
-        "symfony/symfony": "~2.3",
-        "sonata-project/admin-bundle": "dev-master",
+        "symfony/framework-bundle": "~2.3",
+        "symfony/twig-bundle": "~2.3",
+        "symfony/security-core": "~2.4",
```
Comment (Soullivaneuh) [inline composer.json]: Should be `~2.3`.

File: composer.json [hunk 2]
```diff
@@ -17,12 +17,16 @@
         }
     ],
     "require": {
-        "symfony/symfony": "~2.3",
-        "sonata-project/admin-bundle": "dev-master",
+        "symfony/framework-bundle": "~2.3",
```
Comment (Soullivaneuh) [inline composer.json]: Do we need all `symfony/framework-bundle` package?

File: composer.json (hunk 1 above)
Comment (hason) [inline composer.json]: Version ``~2.3`` doesn't exist.

Comment (Soullivaneuh) [inline composer.json]: For what is this package needed?

Comment (hason) [inline composer.json]: https://github.com/sonata-project/SonataDashboardBundle/blob/master/Controller/DashboardAdminController.php#L17

Comment (Soullivaneuh) [inline composer.json]: Please require `symfony/security` instead for SF 2.3 compat.

Comment (rande) [inline composer.json]: @Soullivaneuh the bundle is new. So no need to support SF 2.3 IMO, we can jump to SF2.7

Comment (Soullivaneuh) [inline composer.json]: Why not.

In this case, commit comment and PR should be updated.

**Comment by core23** [comment]
Any news @hason

**Comment by core23** [comment]
Please rebase @hason

**Comment by Soullivaneuh** [comment]
According to the new [Sonata version management](https://github.com/sonata-project/SonataAdminBundle/issues/3053) and next major [release plan](https://github.com/sonata-project/SonataAdminBundle/issues/3731), this project has been refactored regarding branching and versioning.

If you see this message, your PR concerns a patch or a minor release and is not targeting the right branch.

So I'm closing this one, but don't see it as a refusal. If you think your work is still relevant and want to continue, feel free to reopen it on the right branch (e.g. the default one).

Regards.

**Pull request closed without merging**